bin/install-php-phpunit.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e -x
```

Comment:
Can we use `set` for this instead? It breaks locally so makes things hard to test locally 🤷‍♂️

Reply:
gone now, we're back to building and using php5.2 with the latest commit